src/components/Tutorial/index.tsx

```diff
@@ -168,7 +179,13 @@ const Tutorial: FC = () => {
             cursorHeadValue.toLowerCase() === TUTORIAL_CONTEXT[tutorialChoice].toLowerCase())) &&
         gesture() ? (
           <div className='tutorial-trace-gesture'>
-            <GestureDiagram path={gesture()!} size={160} strokeWidth={10} arrowSize={5} className='animate-pulse' />
+            <GestureDiagram
+              path={gesture()!}
```

Comment:
This comment is not blocking, and likely out of scope for this pull request. However, would it be safer to assign a variable such as `const path = gesture()` so that we can call the function once* and not need to include the `!` assertion here (instead, `path={path}`)?

\* I see that we are using the `once` utility to memoize the result and avoid doing the calculation multiple times per render, but would that still serve a purpose if we simply computed the `path` variable once during the render? As it is now, it seems like the `once` utility and `!` assertion are somewhat codependent and could be simplified by straightforwardly assigning the `path` variable once.